tests/sentry/api/endpoints/test_group_details.py

```diff
@@ -464,7 +464,7 @@ def test_assign_id_via_api_key(self):
             url,
             data={"assignedTo": self.user.id},
             format="json",
-            HTTP_AUTHORIZATION="Basic " + b64encode(u"{}:".format(api_key.key)),
+            HTTP_AUTHORIZATION=b"Basic " + b64encode(u"{}:".format(api_key.key).encode("utf-8")),
```

Comment:
```suggestion
            HTTP_AUTHORIZATION=u"Basic " + b64encode(u"{}:".format(api_key.key).encode("utf-8")),
```

https://github.com/getsentry/sentry/pull/20261/files#r471813220

Reply:
Can't do that, can't do u"test" + b"test"

Reply:
Oh wait, this is annoying...

Reply:
I'll fix the formatting one's above to be consistent with these.